### PR TITLE
Update persistent_failures_analysis notebook to match template

### PR DESCRIPTION
**Pull request by isabelizimm**
## Related Issues and Dependencies

 #184

## This introduces a breaking change

- [ ] Yes
- [X] No

## Description

Matched notebook to template. Added `pyarrow` dependency, added "Push to Ceph," "Push to Disk," and "Conclusion." (Please ignore the fact I accidentally committed/reverted the wrong file, which is why there are 3 commits.)

### CODE REVIEW

**Comment by isabelizimm** [comment]
/assign @michaelclifford 
Looks like it passes all checks now : )

File: notebooks/data-sources/TestGrid/metrics/persistent_failures_analysis.ipynb [hunk 1]
```diff
@@ -17,7 +17,7 @@
   },
```
Comment (MichaelClifford) [inline notebooks/data-sources/TestGrid/metrics/persistent_failures_analysis.ipynb:17]: <p>Can you add the linked issues here? I think its #116. So adding the below to the markdown cell should work: </p><p> </p><p> </p><p>"[related issue #116](https://github.com/aicoe-aiops/ocp-ci-analysis/issues/116)"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/201/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (MichaelClifford) [inline notebooks/data-sources/TestGrid/metrics/persistent_failures_analysis.ipynb:17]: <p>We should update the text to explain what this specific calculation is doing. I think the above is just a general place holder text from the template. </p><p> </p><p>Explain how we are generating the Persistent Failure values. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/201/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

**Comment by MichaelClifford** [comment]
@isabelizimm looks good :+1: Can you squash the commits 5 commits into 1. Then we can merge this : )

**Pull request merged**